cfpb/cfgov-refresh
# Add Feedback module

Adds the Feedback module to Browse, Browse Filterable, Landing, Sublanding, Sublanding Filterable, Learn, and Document Detail pages.

## Additions

- Add Feedback model and form
- Add blocks.py for new module
- Add django admin registration for Feedback
- Add feedback template
- Add migration file for Feedback table
- Add handlers module in v1 app to handle module response

## Testing

- `cfgov/manage.py migrate`
- `tox`
- Go to one of the referenced page types and create or edit a page.
 - Select a feedback block
 - Publish the page
- Go to the page's URL and click the Submit feedback button with no content. You should receive an error telling you to select a choice.
- Resubmit with a choice and/or optional feedback
- Go to the Django admin
- Go to the Feedbacks link
- Note the list of feedbacks with a filter option on the right with all of the names of the pages

## Review

- @Scotchester 
- @cfpb/cfgov-backends 

## Screenshots


## Checklist
